daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v2/ScriptF.scala

```diff
@@ -1205,7 +1205,7 @@ object ScriptF {
       case ("SubmitTree", 1) => parseSubmit(v, stackTrace).map(SubmitTree(_))
       case ("TrySubmit", 1) => parseSubmit(v, stackTrace).map(TrySubmit(_))
       case ("TrySubmitConcurrently", 1) => parseSubmitConcurrently(v, stackTrace)
-      case ("Query", 1) => parseQuery(v)
+      case ("QueryACS", 1) => parseQueryACS(v)
```

Comment:
the bug fix was `s/"Query"/"QueryACS"/`, all the other renamings in this file are only for consistency

Reply:
How on earth did the integration tests not catch this...

Reply:
It was caught by repl-functests, some by _some_ integration test at least.

Reply:
somehow none of our `daml-test-files` called `query`!